filesystem/src/main/java/org/frankframework/filesystem/AbstractFileSystemListener.java

```diff
@@ -330,29 +339,11 @@ public Message extractMessage(@Nonnull RawMessageWrapper<F> rawMessage, @Nonnull
 
 	public @Nonnull Map<String, Object> extractMessageProperties(@Nonnull F rawMessage, @Nullable String originalFilename) throws ListenerException {
```

Comment:
I refactored this method a bit to make Sonar happier, but functionally nothing has changed.